crates/pessimistic-proof/src/multi_batch_header.rs

```diff
@@ -58,6 +58,14 @@ where
     // TODO: benchmark if BTreeMap is the best choice in terms of SP1 cycles
     pub imported_local_exit_roots: BTreeMap<NetworkId, H::Digest>,
```

Comment:
I think this is no longer required since LERs are checked against the imported RER.

Reply:
Yeah true, removed